apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx

```diff
@@ -1064,26 +1066,34 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
                             </div>
                         )
                     }
-                    <div className="pt-0 mt-0">
-                        <Field
-                            data-componentid={ `${ testId }-fapi-app-checkbox` }
-                            name={ "isFAPIApp" }
-                            required={ false }
-                            type="checkbox"
-                            value={ [ "isFAPIApp" ] }
-                            children={ [
-                                {
-                                    label: t("console:develop.features.applications.forms.generalDetails.fields" +
-                                        ".isFapiApp.label" ),
-                                    value: "fapiApp"
-                                }
-                            ] }
-                        />
-                        <Hint compact>
-                            { t("console:develop.features.applications.forms.generalDetails.fields" +
-                                ".isFapiApp.hint" ) }
-                        </Hint>
-                    </div>
+                    {
+                        // The FAPI App creation checkbox is only present in OIDC Standard-Based apps
+                        customApplicationProtocol === SupportedAuthProtocolTypes.OAUTH2_OIDC &&
```

Comment:
We can improve formatting here.

customApplicationProtocol === SupportedAuthProtocolTypes.OAUTH2_OIDC
&& selectedTemplate?.name === ApplicationTemplateNames.STANDARD_BASED_APPLICATION
&& isFAPIAppCreationEnabled
&& (

Reply:
fixed with [db451e6](https://github.com/wso2/identity-apps/pull/4616/commits/db451e67d57f674883d760ba9b0831bb14accd8f)